subprojects/diagnostics/src/integTest/groovy/org/gradle/api/tasks/diagnostics/DependencyReportTaskIntegrationTest.groovy

```diff
@@ -804,4 +804,36 @@ compile
 \\--- org.original:original:1.0 -> org.other:another:0.1
 """
     }
+
+    void "doesn't fail if a configuration is not resolvable"() {
```

Comment:
should add some coverage for using `--configuration` on a configuration that isn't resolvable.